src/tcgwars/logic/impl/gen3/PopSeries5.groovy

```diff
@@ -340,6 +313,11 @@ public enum PopSeries5 implements LogicCardInfo {
         pokePower "Purple Ray", {
           text "Once during your turn, when you put Espeon * from your hand onto your Bench, you may use this power. Each Active Pokémon (both yours and your opponent's) is now Confused."
           actionA {
+            if (it==PLAY_FROM_HAND && opp.hand && confirm("Use Purple Ray?")) {
```

Comment:
Shouldn't this be in an onActivate? I think that's how UF Umbreon does it's PokePower.

Edit: and how the Umbreon here does too it seems. :laughing: